pkg/controller/etcdproxy/certs.go

```diff
@@ -183,7 +183,7 @@ func (c *EtcdProxyController) updateAPIServerServingCAConfigMaps(etcdstorage *et
 
 		if val, ok := caConfigMapCopy.Annotations[annCertificateGenerated]; ok == true {
```

Comment:
no need for `== true`. Doesn't govet complain?

Reply:
Fixed in #53